clusterloader2/pkg/measurement/common/generic_query_measurement_test.go

```diff
@@ -147,6 +148,10 @@ func TestGather(t *testing.T) {
 			for _, s := range tc.notWantedData {
 				assert.False(t, strings.Contains(content, s), "summary contains extra data: got: %v, want: no %v", content, s)
 			}
+			testNameLabelPattern := fmt.Sprintf("\"labels\":\\s*\\{\\s*\"MetricName\":\\s*\"%v\"\\s*\\}", tc.name)
+			if match, _ := regexp.MatchString(testNameLabelPattern, content); !match {
```

Comment:
Do not ignore error.

Proposed fix:
* move regexp to some global var (or top variable in this test) and use testNameLabelPattern :=  regexp.MustParse()
* here use testNameLabelPattern.MatchString which returns no error: https://pkg.go.dev/regexp#Regexp.MatchString

Reply:
I don't think I can move regexp to global variable as test case name is part of the pattern.
Added error check.